cmd/main.go

```diff
@@ -114,6 +114,7 @@ func main() {
 		pprofBindAddress              string
 		leaderElectionNamespace       string
 		enableSveltosCtrl             bool
+		disabledProviderTemplates     comaSeperated
```

Comment:
IMO: this is so bad UX having a separate args flag for that

Reply:
any ideas on an alternative?

Reply:
- annotation of some object
- label on some object
- Dedicated ConfigMap with a list of disabled providers

Personally I would prefer annotations

Reply:
Sounds good. Maybe annotate the provider template object itself then?

Reply:
> Maybe annotate the provider template object itself then

If that works out, can be used together with Helm values to disable providers during first install

Reply:
Thanks for the suggestion. I've updated the PR to use the annotation approach.